ecf/hurricane/plots/jevs_hurricane_global_det_tropcyc_plots.ecf

```diff
@@ -33,7 +33,6 @@ module load grib_util/$grib_util_ver
 module load prod_util/${produtil_ver}
 module load prod_envir/${prodenvir_ver}
 module load imagemagick/${imagemagick_ver}
-
 module list
```

Comment:
Add a blank line between `module load imagemagick/${imagemagick_ver}` and `module list`.